Source/WebCore/editing/cocoa/FontAttributesCocoa.mm

```diff
@@ -86,8 +86,8 @@
 RetainPtr<NSDictionary> FontAttributes::createDictionary() const
 {
     NSMutableDictionary *attributes = [NSMutableDictionary dictionary];
-    if (auto cocoaFont = font ? (__bridge CocoaFont *)font->getCTFont() : nil)
-        attributes[NSFontAttributeName] = cocoaFont;
+    if (RetainPtr cocoaFont = font ? (__bridge CocoaFont *)font->getCTFont() : nil)
```

Comment:
Can we use `bridge_cast` here while we're at it? You probably need to add CTFont / NSFont entry to `Source/WTF/wtf/cocoa/TollFreeBridging.h` though